src/iqm/qiskit_iqm/move_gate.py

```diff
@@ -55,8 +59,7 @@ def _define(self):
 
         q = QuantumRegister(2, "q")
         qc = QuantumCircuit(q, name=self.name)
```

Comment:
Attempt to keep track of a potential given label.
```suggestion
        qc = QuantumCircuit(q, name=self.label if self.label else self.name)
```